pkg/manifest/manifest_test.go

```diff
@@ -12,50 +14,50 @@ import (
 )
 
 func TestReadManifest_URL(t *testing.T) {
-	c := NewTestConfig(t)
-	url := "https://raw.githubusercontent.com/deislabs/porter/master/pkg/config/testdata/simple.porter.yaml"
-	m, err := c.ReadManifest(url)
+	c := config.NewTestConfig(t)
+	url := "https://raw.githubusercontent.com/deislabs/porter/v0.17.0-beta.1/pkg/config/testdata/simple.porter.yaml"
```

Comment:
This file moved in the refactoring so I am using a tag for now to prevent the build from breaking when the PR is merged.